projects/shared-models/mini-user-profile.model.ts

```diff
@@ -0,0 +1,15 @@
+import { IBadge } from './badge.model';
+import { ITag } from './tag.model';
+
+export interface IMiniUserProfile {
```

Comment:
Let's extend the `IUser` model instead of declaring a new model

Reply:
done